plugins.html

```diff
@@ -360,6 +360,11 @@ <h1>Kanboard Plugins</h1>
                     <p>Use the Mailgun API to send emails and create tasks from emails.</p>
                     <p><em>By Frédéric Guillot</em></p>
                 </dd>
+                <dt><a href="https://github.com/creecros/MarkdownPlus">Encrypted Content</a></dt>
```

Comment:
The title seems wrong. Bad copy and paste?

Reply:
Ya, it started out as just emojis then I got bored and added more stuff, changed the name, yadda yadda. I'll change it in a minute

Reply:
all set